__tests__/pathfinding.service.test.js

```diff
@@ -1,12 +1,13 @@
 import * as testData from "./__testData__/pathfinding.data";
 import { POIInfo, buildingFloors } from "../src/constants";
-import { BuildingId } from "../src/types/main";
+import * as constants from "../src/constants";
+import { BuildingId, POICategory } from "../src/types/main";
 import PathFindingService from "../src/services/pathfinding.service";
 
-describe("Find the shortest path on a given floor", () => {
+describe("Find the shortest path between two locations/pois", () => {
```

Comment:
Same as Ragith's comment, `POIs` instead of `pois`
Just CTRL+F for lowercase poi and replace them with uppercase POI